lib/slurp/logs/subscriptions.ex

```diff
@@ -77,4 +79,20 @@ defmodule Slurp.Logs.Subscriptions do
   def put(log_subscription, store_id \\ Logs.LogSubscriptionStore.default_store_id()) do
     Logs.LogSubscriptionStore.put(log_subscription, store_id)
   end
+
+  defp parse_event_signature(signature) when is_bitstring(signature), do: parse_event_signature({signature, []})
+  defp parse_event_signature({event_header, address}), do: {event_header, address}
+
+  defp make_log_subscription(%{address: []} = attrs) do
+    make_log_subscription(%{attrs | address: ""})
```

Comment:
I feel like this should be `nil`. Was `nil` causing a problem with `ETS`?

Reply:
No; I had just used empty string because it was convenient at the time; I can go back and make the nullcase nil